src/AppInstallerCommonCore/MSStoreDownload.cpp

```diff
@@ -910,6 +911,13 @@ namespace AppInstaller::MSStore
             {
                 SFS::RequestParams sfsClientRequest;
                 sfsClientRequest.productRequests = { {std::string{ wuCategoryId }, {}} };
+                if (AppInstaller::Settings::Network().GetProxyUri())
+                {
+                    const auto& proxyUri = AppInstaller::Settings::Network().GetProxyUri().value();
+
+                    sfsClientRequest.proxy = proxyUri;
+                    AICLI_LOG(Core, Info, << "Passing proxy to SFS client " << proxyUri);

```

Comment:
nit nit: we usually log before the action if log is "Passing ...", please either move this one line above or change it to be "passed ..."

Reply:
Fixed, thanks Yao